openlcs/libs/kojiconnector.py

```diff
@@ -369,6 +369,43 @@ def download_container_image_archives(self, build, dest_dir, arch=None):
             else:
                 raise ValueError("No images found.") from None
 
+    def download_oci_remote_source_archives(self, component, dest_dir):
+        """
+        Find container remote source tar file and Download from Brew/Koji.
+        """
+        all_archives = self._service.listArchives(int(component["build_id"]))
+        if not all_archives:
+            raise ValueError("No build archives found.") from None
+
+        # find remote source tar file name
+        filename = ""
+        for archive in all_archives:
+            if archive.get('btype') == 'remote-sources' and\
+                    archive.get('type_name') == 'tar':
+                filename = archive.get("filename")
+                break
```

Comment:
There may be several remote source tarballs for a container , and all tarballs should be downloaded without knowing which remote source tarball is the one needed. e.g., https://brewweb.engineering.redhat.com/brew/buildinfo?buildID=2119685

Reply:
Thanks. I will change the code